static/docs/commands-reference/get-url.md

```diff
@@ -0,0 +1,166 @@
+directory.
+
+It's important to note that this command does not require an initialized
+repository to work in. It's a single-purpose command that can be used out of the
+box after installing DVC.
+
+DVC supports several types of (local or) remote locations (protocols):
+
+| Type    | Discussion                                              | URL format                                 |
+| ------- | ------------------------------------------------------- | ------------------------------------------ |
+| `local` | Local path                                              | `/path/to/local/file`                      |
+| `s3`    | Amazon S3                                               | `s3://mybucket/data.csv`                   |
+| `gs`    | Google Storage                                          | `gs://mybucket/data.csv`                   |
+| `ssh`   | SSH server                                              | `ssh://user@example.com:/path/to/data.csv` |
+| `hdfs`  | HDFS                                                    | `hdfs://user@example.com/path/to/data.csv` |
+| `http`  | HTTP to file with _strong ETag_ (see explanation below) | `https://example.com/path/to/data.csv`     |
+
+> `remote://myremote/path/to/file` notation just means that a DVC
+> [remote](/doc/commands-reference/remote) `myremote` is defined, and when DVC
+> is running it internally expands this URL into a regular S3, SSH, GS, etc URL
+> by appending `/path/to/file` to the `myremote`'s configured base path.
+
+Another way to understand the `dvc get-url` command is as a tool for downloading
+data files.
+
+On GNU/Linux systems for example, instead of `dvc get-url` with HTTP(S) it's
+possible to instead use:
+
+```dvc
+$ wget https://example.com/path/to/data.csv
+```
+
+## Options
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+<details>
+
+### Click and expand for a local example
+
+```dvc
+$ dvc get-url /local/path/to/data
+```
+
+The above command will copy the `/local/path/to/data` file or directory into
+`./dir`.
+
+</details>
+
+### Click for AWS S3 example
+
+This command will copy an S3 object into the current working directory with the
+same file name:
+
+```dvc
+$ dvc get-url s3://bucket/path
+```
+
+By default DVC expects your AWS CLI is already
+[configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+DVC will be using default AWS credentials file to access S3. To override some of
+these settings, you could the options described in `dvc remote modify`.
+
+We use `boto3` library to set up a client and communicate with AWS S3. The
+following API methods are performed:
+
+- `list_objects_v2`, `list_objects`
+- `head_object`
+- `download_file`
+- `upload_file`
```

Comment:
upload, delete, copy are not performed for sure, it might be that we only need the read permissions for the key

Reply:
Hey yeah why does `get-url` need to upload, delete, or copy anything within S3? I've removed the writing operations and permissions from this doc for now... Not making sense to me, can you confirm @efiop ? Thanks

Reply:
@efiop please, can you provide more context here.

Reply:
Finally figured this out. See commit 9d0819d with message "s3: update info on boto3 methods and permissions required..."